make_files.py

```
@@ -0,0 +1,37 @@
+import os
+import re
+
+with open('/tmp/telemetry.rs', 'r') as f:
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/telemetry.rs"

(S108)